### PR TITLE
Applications API working for Get, Post, Put, Get {id} and Delete

### PR DESCRIPTION
**Pull request by jdepaul**

### CODE REVIEW

File: src/K9OCRS/Models/Applications.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace K9OCRS.Models
+{
+    public class Applications
```
Comment (deznorth) [inline src/K9OCRS/Models/Applications.cs:8]: What is this model for? On the Controller only the `ClassApplication` entity and the `ApplicationsListRequest` model are being used, so this file can be deleted. Unless you're planning on using it in the future, in which case it should be moved to the `Models/ApplicationsManagement` folder.

File: src/K9OCRS/Models/Class.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace K9OCRS.Models
+{
+    public class Class
```
Comment (deznorth) [inline src/K9OCRS/Models/Class.cs:8]: This is just an empty file. It should be deleted

**Review by deznorth — CHANGES_REQUESTED** [review]
Just got one empty file to delete and one that needs either moving it, and explaining what it might be used for in the future, or deleting it if it was not intentionally left there.

File: src/K9OCRS/Models/Applications.cs (hunk 1 above)
Comment (jdepaul) [reply src/K9OCRS/Models/Applications.cs:8]: Probably left over from when I was starting on the api.

**Review by deznorth — APPROVED** [review]

**Pull request merged**